libr/anal/p/anal_x86_cs.c

```diff
@@ -267,6 +267,7 @@ static int analop(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len) {
 		}
 		switch (insn->id) {
 		case X86_INS_FNOP:
+			op->family = R_ANAL_OP_FAMILY_FPU;
```

Comment:
no srsly, this is an FPU nop. and if you read the code you'll discover that it's already set to type=NOP